<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QWidget" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainwindow</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>1711</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>1025</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>800</width>
     <height>1031</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>200</number>
   </attribute>
   <column>
    <property name="text">
     <string>Date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Next Day</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Differ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Result</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="previousDayButton">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>40</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous Day</string>
   </property>
  </widget>
  <widget class="QPushButton" name="isLeapButton">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>100</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Is Leap</string>
   </property>
  </widget>
  <widget class="QPushButton" name="birthdayButton">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>220</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Till Your Birthday</string>
   </property>
  </widget>
  <widget class="QPushButton" name="weekNumberButton">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>160</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Week Number</string>
   </property>
  </widget>
  <widget class="QPushButton" name="durationButton">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>280</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Duration</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>40</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Your birthday:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBirthday">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>40</y>
     <width>541</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>70</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Days till birthday:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDaysTillBirthday">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>70</y>
     <width>321</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>110</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimumDate">
    <date>
     <year>1754</year>
     <month>9</month>
     <day>1</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>400</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Date</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeButton">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>460</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Date</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateBirthdayEdit">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>530</y>
     <width>110</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimumDate">
    <date>
     <year>1754</year>
     <month>9</month>
     <day>1</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addBirthdayButton">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>580</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Birthday Date</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>970</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
